pypesto/optimize/optimizer.py

```diff
@@ -720,6 +725,84 @@ def is_least_squares(self):
         return False
 
 
+class PyswarmsOptimizer(Optimizer):
+    """
+    Global optimization using pyswarms.
+
+    Parameters
+    ----------
+    options:
+        Optimizer options that are directly passed on to pyswarms.
+        c1: cognitive parameter
+        c2: social parameter
+        w: inertia parameter
+    """
+
+    def __init__(self, par_popsize: float = 10, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 1000, 'c1': 0.5, 'c2': 0.3, 'w': 0.9}
```

Comment:
Is it necessary to specify c1, c2, w? If not, I would prefer to has pyswarms choose its own defaults. maxiter does not seem to be a valid parameter.

Reply:
better specify iters as a separate parameter?

Reply:
1. The above written parameter values for c1,c2,w are the defaults, taken directly from pyswarms api. I would leave them there so one has the option to change them

2. I assign the value of 'maxiter' to iters by ```iters=self.options['maxiter']```. Do you want me to rename 'maxiters' into 'iters_value'?